tests/integ/test_dataframe.py

```diff
@@ -3352,6 +3356,163 @@ def test_append_existing_table(session, local_testing_mode):
         Utils.drop_table(session, table_name)
 
 
+@pytest.mark.xfail(
+    "config.getoption('local_testing_mode', default=False)",
+    reason="Dynamic table is a SQL feature",
+    run=False,
+)
```

Comment:
should we use `@pytest.mark.skipif` here instead? This would mean the test is skipped in localtesting mode and run in other integ tests